cpp/test/Slice/errorDetection/IllegalUseOfKeyword.ice

```diff
@@ -16,22 +14,21 @@ module Test
         int i;
     }
 
-    class interface;
+    class idempotent;
     class struct
     {
-        // void f();
         int i;
     }
 
-    // interface local;
```

Comment:
Another place where we're supposed to be checking keywords as identifiers.
Instead of updating this file when we removed `local`, `implements`, and `LocalObject`, we just commented out 2 of them and ignored the 3rd. We should be using actual keywords in this test.